module/userProjects/helper.js

```diff
@@ -3045,16 +3082,16 @@ module.exports = class UserProjectsHelper {
 						return resolve(programAndSolutionInformation)
 					}
 
-					if (
-						libraryProjects.data['entityInformation'] &&
-						libraryProjects.data['entityInformation'].entityType !==
-							programAndSolutionInformation.data.solutionInformation.entityType
-					) {
-						throw {
-							message: CONSTANTS.apiResponses.ENTITY_TYPE_MIS_MATCHED,
-							status: HTTP_STATUS_CODE.bad_request.status,
-						}
-					}
+					// if (
+					// 	libraryProjects.data['entityInformation'] &&
+					// 	libraryProjects.data.entityType !==
```

Comment:
@praveenKDass  check entityType is required in template use same entityType in project also